### ISSUE TITLE
Docker Entrypoint Command "configure" Broken

### ISSUE THREAD

**Issue by bryonbean** [Kitware/CDash#887]
`docker-compose run --rm cdash configure`

 run with the default values in the docker-compose.yml file results in the following output:
```
127.0.0.1 - - [17/Sep/2019:15:50:06 +0000] "GET /ajax/findusers.php?search=jdoe@acme.com HTTP/1.1" 200 906
127.0.0.1 - - [17/Sep/2019:15:50:06 +0000] "POST /manageUsers.php HTTP/1.1" 200 8374
127.0.0.1 - - [17/Sep/2019:15:50:06 +0000] "GET /ajax/findusers.php?search=jdoe@acme.com HTTP/1.1" 200 904
Warning: unable to create user account jdoe@acme.com: unknown error
127.0.0.1 - - [17/Sep/2019:15:50:07 +0000] "GET /ajax/findusers.php?search=admin@example.org HTTP/1.1" 200 906
127.0.0.1 - - [17/Sep/2019:15:50:07 +0000] "POST /manageUsers.php HTTP/1.1" 200 8368
127.0.0.1 - - [17/Sep/2019:15:50:07 +0000] "GET /ajax/findusers.php?search=admin@example.org HTTP/1.1" 200 902
Warning: unable to create user account admin@example.org: unknown error
127.0.0.1 - - [17/Sep/2019:15:50:08 +0000] "GET /ajax/findusers.php?search=donotreply@example.org HTTP/1.1" 200 910
127.0.0.1 - - [17/Sep/2019:15:50:08 +0000] "POST /manageUsers.php HTTP/1.1" 200 8368
127.0.0.1 - - [17/Sep/2019:15:50:08 +0000] "GET /ajax/findusers.php?search=donotreply@example.org HTTP/1.1" 200 904
Warning: unable to create user account donotreply@example.org: unknown error
127.0.0.1 - - [17/Sep/2019:15:50:09 +0000] "GET /ajax/findusers.php?search=strange.name@gmail.com HTTP/1.1" 200 902
127.0.0.1 - - [17/Sep/2019:15:50:09 +0000] "POST /manageUsers.php HTTP/1.1" 200 8374
127.0.0.1 - - [17/Sep/2019:15:50:09 +0000] "GET /ajax/findusers.php?search=strange.name@gmail.com HTTP/1.1" 200 906
Warning: unable to create user account strange.name@gmail.com: unknown error
127.0.0.1 - - [17/Sep/2019:15:50:10 +0000] "GET /ajax/findusers.php?search=remove.this.user@example.org HTTP/1.1" 200 906
Warning: could not remove user remove.this.user@example.org: user not found
```

This has likely been broken because of the move to Laravel.

**Comment by RedDwarf69** [comment]
I have the same problem
```
$ docker images | grep -e kitware -e mysql
kitware/cdash                                     latest              49267f56f2ff        8 days ago          1.07GB
mysql/mysql-server                                5.5                 1105404428e8        8 months ago        173MB
$ git rev-parse HEAD
046d49808a7b8cc74a191810e4a16926cdc5568c
```

**Comment by zackgalbreath** [comment]
This was fixed in #986. Please feel free to reopen this issue if it still doesn't work for you.
